scout/server/blueprints/login/controllers.py

```diff
@@ -1,34 +1,49 @@
-# -*- coding: utf-8 -*-
+"""Code for login controllers"""
+
+DETECTIVES = [
```

Comment:
There is something really wrong in this list: where are Jessica Fletcher, Inspector Clouseau, and Columbo????

Reply:
I'll add them! These are just for future inspiration

Reply:
I thought we could have some kind of Easter egg so that when in a certain range of events you become one of those